content/kapp-controller/docs/latest/package-consumption.md

```diff
@@ -18,15 +25,15 @@ kapp-controller needs to know which packages are available to install. One way t
 apiVersion: install.package.carvel.dev/v1alpha1
 kind: PackageRepository
 metadata:
-  name: basic.test.carvel.dev
+  name: simple-package-repository
 spec:
   fetch:
     image:
       url: k8slt/kctrl-pkg-repo:v1.0.0
```

Comment:
did we change contents of k8slt/kctrl-pkg-repo:v1.0.0 to include simple-app.corp.com packages? also can we rename this to k8slt/corp-com-pkg-repo:1.0.0?

Reply:
The packages were renamed to `simple-app.corp.com` and pushed. But yes, I can rename the repo image as well to `corp-com-pkg-repo:1.0.0`.

Reply:
I pushed `corp-com-pkg-repo:1.0.0` and updated image used for PackageRepository to `k8slt/corp-com-pkg-repo:1.0.0`.